src/redux/modules/favorites/fave-stop-reducer.js

```diff
@@ -0,0 +1,118 @@
+import {
+  REMOVE_LOCAL_STOP,
+  TOGGLE_FAVE_STOP,
+  FETCH_FAVE_REQUEST,
+  FETCH_FAVE_SUCCESS,
+  FETCH_FAVE_FAILURE
+} from './fave-stop-actions'
+
+
+//@app.route('/api/v2/stops/<stopId>')
+
+const initialState = {
+  faveStopIds: getFaveStops(), //from localStorage
+  faveStops: []
+}
+
+/*
+ * @name setFaveStops
+ * @param {array} faveStops
+ * @description
+ * Replaces the contents of localStorage['faveStops']
+ * with a new array.
+ */
+
+function setFaveStops(faveStopIds) {
+  localStorage['faveStops'] = faveStopIds
+}
+
+/*
+ * @name getFaveStops
+ * @description
+ * Returns an array of stop ids that are favorited.
+ */
+
+function getFaveStops() {
+  const faveStopIds = localStorage['faveStops']
+  return faveStopIds ? faveStopIds.split(',') : []
+}
+
+export default function faveStops(state = initialState, action) {
+
+  switch (action.type) {
+
+    case TOGGLE_FAVE_STOP:
+      const index = state.faveStopIds.indexOf(action.stopId)
+      const indexOfFave = state.faveStops.map(function(e) {
+        return e.stopId
+      }).indexOf(action.stopId)
```

Comment:
This is a clever use of Array#map, but in the future you can get away with just using `lodash#findIndex` which does basically the same thing.